oauth2client/django_util/decorators.py

```diff
@@ -0,0 +1,50 @@
+from django.shortcuts import redirect
+from oauth2client.django_util.storage import get_storage
+from six import wraps
+from . import settings
+
+
+def _build_scope_query_param(scopes):
```

Comment:
Amazingly Django does not have a built-in function for this, despite finding about 10 people re-creating their own. Ended up tweaking one I found and adding it so it's more general at least.